py-polars/src/series/mod.rs

```diff
@@ -342,6 +342,12 @@ impl PySeries {
     ) -> PyResult<PySeries> {
         let series = &self.series;
 
+        if output_type.is_none() {
+            polars_warn!(
+                "calling `map_elements` without specifying `return_dtype` can lead to unpredictable results, \
+                please specify `return_dtype` to silence this warning")
```

Comment:
For warnings we want to use full punctuation:

```suggestion
                "Calling `map_elements` without specifying `return_dtype` can lead to unpredictable results. \
                Specify `return_dtype` to silence this warning.")
```